src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -165,6 +189,29 @@ export class PrintableUri extends Component {
     ).el;
   }
 
+  private buildEllipsis(action: (e: Event) => void) {
+    const button = $$(
+      'span',
+      {
+        ariaLabel: l('CollapsedUriParts')
+      },
+      '...'
+    );
+    const element = $$(
+      'span',
+      {
+        className: 'coveo-printable-uri-ellipsis',
+        role: 'listitem'
+      },
+      button
+    ).el;
+    new AccessibleButton()
```

Comment:
What about making the `button` element a `<button>` instead of a `<span>`? You wouldn't need "make it accessible" it would be out of the box IIRC.

Reply:
Good point, I'll replace it with a button.